modules/ROOT/pages/design-create-publish-api-raml-editor.adoc

```diff
@@ -10,21 +10,29 @@ Within Anypoint Platform, your user ID must be assigned the _Design Center Devel
 
 == Procedure
 
+. Open Design Center from the homepage of Anypoint Platform.
 . Create an API-specification project that you can work in.
  .. On the *Projects* page in Design Center, click *Create* and then click *Create API Specification*.
- .. In the *New Specification* section of the dialog that is opened, name your project and select the option *Text editor* to use the text editor.
+ .. In the *New Specification* section of the dialog that is opened, name your project. You can change the name (but not the extension) later, if you want to.
```

Comment:
```suggestion
 .. In *New Specification*, name your project. 
 You can change the name (but not the extension) later, if you want to.
```

Reply:
@kbostrom , I disagree with rendering the text vague.